marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/steps/MatchingStepController.java

```diff
@@ -75,6 +75,14 @@ public ResponseEntity<JsonNode> getExclusionList(@PathVariable String listName)
         return ResponseEntity.ok(ArtifactService.on(getHubClient().getFinalClient()).getArtifact("exclusionList", listName));
     }
 
+    @RequestMapping(value = "/exclusionList/{listName}", method = RequestMethod.DELETE)
+    @ResponseBody
+    @ApiOperation(value = "Get an exclusion list")
```

Comment:
Here it should be "Delete an exlusion list"

Reply:
Updated the description. Thanks!